### ISSUE TITLE
Send reward action does not seem to function.

### ISSUE THREAD

**Issue by mibby** [Efnilite/Walk-in-the-Park#88]
**Describe the bug**
Using the send reward action doesn't seem to work under 1.19.2.

**To Reproduce**
```
interval-rewards:
  50:
    - "vault:25"
    - "send:<green>+25 Coins"
```

1. Jump 50 blocks.
2. No message?

**Errors**
None that I can see.

**Versions**
- IP v3.8.0
- vilib v1.0.11
- Paper dev 125 (1.19.2)

**Additional context**
This use to work but now doesn't. Broke due to chat changes in 1.19+?

**Comment by Efnilite** [comment]
This bug occurs due to a recent change in v3.8.0. You can use 3.70 to fix it. A build fixing this will be released when I come back from holiday, in about 10 days :)

**Comment by Efnilite** [comment]
Fixed in v3.8.1 -- thanks for reporting <3
